### PR TITLE
feat: catalyst manager

### PR DESCRIPTION
**Pull request by eordano**
Preview at https://catalysts.decentraland.io

### CODE REVIEW

File: status/src/server.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export const server = 'peer.decentraland.org'
```
Comment (nchamo) [inline status/src/server.ts:1]: If we change this to:
```
process.ENV.CATALYST_URL ?? peer.decentraland.org
```
then we can easily run this together with catalysts. 

Take into account that `CATALYST_URL` includes the protocol (http or https)

File: status/src/denyList/Denylist.tsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  if (!chain || new Date(chain.expiration).getTime() < new Date().getTime()) {
+    const account = Account.create()
+
+    const payload = {
+      address: account.address.toString(),
+      publicKey: bufferToHex(account.publicKey),
+      privateKey: bufferToHex(account.privateKey),
+    }
+
+    const ONE_MONTH_MILLISECONDS = 31 * 24 * 60 * 60 * 1000
```
[the remark this answers is not in the document]
Comment (eordano) [reply status/src/denyList/Denylist.tsx:59]: just for this line, I think I'd rather save the dependency

File: status/src/denyList/Denylist.tsx [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+              <input className="input-denylist" name="content" id="deny-content"></input>
+              <button type="submit">Submit</button>
+            </form>
+            <h5>Deny by entity id</h5>
+            <form onSubmit={denyBy('entity', provider, identity)}>
+              <input className="input-denylist" name="entity" id="deny-entity"></input>
+              <button type="submit">Submit</button>
+            </form>
+            <h5>Deny by parcel coordinate</h5>
+            <form onSubmit={denyBy('pointer', provider, identity)}>
```
[the remark this answers is not in the document]
Comment (eordano) [reply status/src/denyList/Denylist.tsx]: Thanks! I'll update this

File: status/src/layout/App.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import { Header } from './Header'
+
+const root = '/'
+const comms = '/comms'
+const denylist = '/denylist'
+const scenes = '/scenes'
+const profiles = '/profiles'
+const dao = '/dao'
+
+export const contentServer = `https://${server}/content/`
```
[the remark this answers is not in the document]
Comment (eordano) [reply status/src/layout/App.tsx]: Fixed

File: status/src/profiles/User.tsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import useSWR from "swr";
+import { fetchJSON } from "../components/fetchJSON";
+import { Loading } from "../components/Loading";
+import { server } from "../server";
+import { Avatar } from "./Avatar";
+
+const commsServer = "https://" + server + "/comms/";
```
[the remark this answers is not in the document]
Comment (eordano) [reply status/src/profiles/User.tsx]: Done

File: status/src/server.ts (hunk 1 above)
Comment (eordano) [reply status/src/server.ts]: Changed to be REACT_APP_CATALYST_URL

**Pull request merged**